cli/src/eval.rs

```diff
@@ -2,39 +2,44 @@ use nickel_lang_core::{eval::cache::lazy::CBNCache, program::Program};
 
 use crate::{
     cli::GlobalOptions,
+    customize::CustomizeMode,
     error::{CliResult, ResultErrorExt},
+    input::{InputOptions, Prepare},
 };
 
 #[derive(clap::Parser, Debug)]
-pub struct EvalCommand {}
+pub struct EvalCommand {
+    #[cfg(debug_assertions)]
+    /// Skips the standard library import. For debugging only. This does not affect REPL
```

Comment:
Now that this has become a flag of `eval` only, I guess you can't even use it with the REPL anymore?

```suggestion
    /// Skips the standard library import. For debugging only
```